src/twisted/conch/ssh/filetransfer.py

```diff
@@ -44,12 +46,13 @@ def dataReceived(self, data):
             data, self.buf = self.buf[5:4+length], self.buf[4+length:]
             packetType = self.packetTypes.get(kind, None)
             if not packetType:
-                log.msg('no packet type for', kind)
+                self.log.info('no packet type for {kind}', kind=kind)
                 continue
             f = getattr(self, 'packet_{}'.format(packetType), None)
             if not f:
-                log.msg('not implemented: {}'.format(packetType))
-                log.msg(repr(data[4:]))
+                self.log.info('not implemented: {packetType}',
+                              packetType=packetType)
+                self.log.info(repr(data[4:]))
```

Comment:
I think this should be merged with the previous message like you did in another place.